src/oemof/tabular/facades/experimental/battery_electric_vehicle.py

```diff
@@ -0,0 +1,354 @@
+    attributes.
+
+    >>> from oemof import solph
+    >>> from oemof.tabular import facades
+    >>> my_bus = solph.Bus('my_bus')
+    >>> my_bev = Bev(
+    ...     label='my_bev',
+    ...     bus=el_bus,
+    ...     carrier='electricity',
+    ...     tech='bev',
+    ...     storage_capacity=1000,
+    ...     capacity=50,
+    ...     availability=[0.8, 0.7, 0.6],
+    ...     drive_power=[0.3, 0.2, 0.5],
+    ...     amount=450,
+    # ...     loss_rate=0.01,
+    ...     initial_storage_level=0,
+    ...     min_storage_level=[0.1, 0.2, 0.15],
+    ...     max_storage_level=[0.9, 0.95, 0.92],
+    ...     efficiency=0.93
+    ...     )
+
+    """
+
+    electricity_bus: Bus
+
+    availability: Union[float, Sequence[float]] = 1
+
+    max_charging_power: Union[float, Sequence[float]] = 0
+
+    drive_power: int = 0
+
+    capacity: int = 0
+
+    storage_capacity: int = 0
+
+    initial_storage_capacity: float = 0
+
+    drive_consumption: Sequence = None
+
+    v2g: bool = False
+
+    transport_commodity_bus: Bus = None
+
+    input_parameters: dict = field(default_factory=dict)
+
+    output_parameters: dict = field(default_factory=dict)
+
+    expandable: bool = False
+
+    lifetime: int = None
+
+    bev_capacity_cost: Sequence[float] = None
+
+    invest_c_rate: Sequence[float] = None
+
+    marginal_cost: float = 0
+
+    balanced: bool = False
+
+    pkm_conversion_rate: float = 1
+
+    efficiency_mob_g2v: float = 1
+
+    efficiency_mob_v2g: float = 1
+
+    efficiency_mob_electrical: float = 1
+
+    efficiency_sto_in: float = 1
+
+    efficiency_sto_out: float = 1
+
+    def build_solph_components(self):
+        # use label as prefix for subnodes
+        self.facade_label = self.label
+        self.label = self.label + "-storage"
+
+        # convert to solph sequences
+        self.availability = solph_sequence(self.availability)
+        self.max_charging_power = solph_sequence(self.max_charging_power)
+
+        # TODO: check if this is correct
+        self.nominal_storage_capacity = self.storage_capacity
+        # self.nominal_storage_capacity = self._nominal_value(
+        #     self.storage_capacity)
+
+        self.balanced = self.balanced  # TODO to be false in multi-period
+
+        # create internal bus
+        internal_bus = Bus(label=self.facade_label + "-bus")
+        self.bus = internal_bus
+        subnodes = [internal_bus]
+
+        # ##### Vehicle2Grid Converter #####
+        if self.v2g:
+            vehicle_to_grid = Converter(
+                label=self.facade_label + "-v2g",
+                inputs={
+                    internal_bus: Flow(
+                        # variable_costs=self.carrier_cost,
+                        # **self.input_parameters
+                    )
+                },
+                outputs={
+                    self.electricity_bus: Flow(
+                        nominal_value=self._nominal_value(
+                            value=self.max_charging_power
```

Comment:
here, you could simply write `nominal_value=self.max_charging_power` instead of calling the function (see comment above)

Reply:
`value=self.max_charging_power if self.expandable else None`